components/service/glean/src/main/java/mozilla/components/service/glean/BooleanMetricType.kt

```diff
@@ -48,4 +49,32 @@ data class BooleanMetricType(
             )
         }
     }
+
+    /**
+     * Tests whether a value is stored for the metric for testing purposes only
+     *
+     * @return true if metric value exists, otherwise false
+     */
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    fun testHasValue(
+        pingName: String =
```

Comment:
To retain the same behaviour as the "default" keyword, I think this could simply default to `defaultStorageDestinations.first()`. This applies to the other default values in this PR as well.